### ISSUE TITLE
"Cannot read property 'processors' of undefined" when using FeatureLayer, Angular 10

### ISSUE THREAD

**Issue by RyannGalea** [Esri/angular-cli-esri-map#74]
I had posted this on the Esri community page and @andygup suggested creating the issue here.

I have a project I have been building for the past year with no apparent issues with the FeatureLayer, and something changed recently I cannot pinpoint what exactly changed because I did not update any packages. 

All of my FeatureLayers give me this error: 
![image](https://user-images.githubusercontent.com/18360468/89727686-4a877000-da6a-11ea-925f-6d52770c8c2b.png)

and I have no idea how to resolve it. I created a repo which is a slimmed-down version of my MapModule and I get the same error. You can clone, install dependencies, and 'ng serve' which should reproduce the issue.

The FeatureLayer in this specific repo is supplied by local government and outlines property boundaries, but this happens to any FeatureLayer I have in my project.

Thanks for taking the time to take a look

Repo: https://github.com/RyannGalea/arcgistest.git

**Comment by andygup** [comment]
Cross reference: https://community.esri.com/thread/257418-featurelayer-typeerror-cannot-read-property-processors-of-undefined

**Comment by andygup** [comment]
I tested both the point feature layer and polygon feature layer with no errors in the current ArcGIS Online MapViewer, MapViewer beta as well as using the code in this repo + Angular 10. See the screenshot below (using Angular 10). 

I suspect the primary problem is a mismatch between your worker API version and the arcgis-webpack-plugin version. These have to be the same. Try changing `map.module.ts' to this:

```
  constructor() {
    const DEFAULT_WORKER_URL        = `https://js.arcgis.com/4.16/` // This version has to match arcgis-webpack-plugin version
    esriConfig.workers.loaderUrl    = `${DEFAULT_WORKER_URL}dojo/dojo-lite.js`
    esriConfig.workers.loaderConfig = LoaderConfig(DEFAULT_WORKER_URL)
  }

```

I also commented out `userDefinedExcludes` in `esri-web.config.js` just to make sure all API modules were bundled. 


![image](https://user-images.githubusercontent.com/510440/89839722-b41b8300-db2b-11ea-9439-f6ed2d248018.png)
